client/src/components/Applet/Applet.js

```diff
@@ -1,28 +1,46 @@
 import React, { Component } from 'react';
-import FontAwesome from 'react-fontawesome';
+// import FontAwesome from 'react-fontawesome';
+import BookWrapper from '../BookWrapper';
+import AvailabilityWrapper from '../AvailabilityWrapper';
+
+const domain = process.env.DOMAIN || 'https://localhost:';
```

Comment:
I would check to see if this line is working as expected. In the browser we don't have process variable